app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayViewModel.kt

```diff
@@ -57,4 +55,14 @@ class NavigationOverlayViewModel(
                     else -> View.NO_ID
                 }
             }
+
+    val toolBarFocus : Observable<Int> = toolbarViewModel.state
+            .map{state ->
+                when {
+                    state.backEnabled -> R.id.navButtonBack
+                    state.forwardEnabled -> R.id.navButtonForward
+                    state.refreshEnabled -> R.id.navButtonReload
+                    else -> R.id.turboButton
```

Comment:
Hm, this is not entirely clear to me.  This emits a new ID any time toolbar state changes.  I see that it's used to set `navUrlInput.nextFocusUpId`, but that isn't really made clear to me by the name `toolBarFocus`.  Could you rename this to better communicate what it's doing (which is emitting the ID of the currently active toolbar button that is furthest to the left)?